.github/workflows/pull-requests.yml

```diff
@@ -54,8 +54,10 @@ jobs:
   Formatting-Linting:
```

Comment:
Wouldn't it be better to do all lint related work in the Linter section? Doing this in a separate section seems unnecessary.

Reply:
Moved the linter check to the linting section in the latest commit.